perun-base/src/main/java/cz/metacentrum/perun/core/api/exceptions/ConsentNotExistsException.java

```diff
@@ -0,0 +1,45 @@
+package cz.metacentrum.perun.core.api.exceptions;
+
+import cz.metacentrum.perun.core.api.Consent;
+
+/**
+ * This exception is thrown when trying to create a consent
```

Comment:
Wrong description. Also, we usually add @ author tags to new classes/exceptions, please add yours.